pkg/plugin/aggregation/aggregator.go

```diff
@@ -311,7 +311,7 @@ func (a *Aggregator) handleResult(result *plugin.Result) error {
 }
 
 func (a *Aggregator) handleArchiveResult(result *plugin.Result) error {
-	resultsDir := path.Join(a.OutputDir, result.Path())
+	resultsDir := filepath.Join(a.OutputDir, result.Path())
```

Comment:
Somewhat preemptively changed these to `filepath`; it will help when we start ensuring compatibility on windows nodes. No change otherwise.